SaveGameEditor/MainForm.cs

```diff
@@ -541,28 +528,38 @@ private DataTable BuildFlagTable()
         {
             DataTable t = new DataTable();
             t.Columns.Add("Key");
-            for (int i = _gameSave.Checkpoints.Count - 2; i >= 0; i--)
+
+            bool first = true;
+            for (int i = _gameSave.Checkpoints.Count - 1; i >= 0; i--)
             {
-                t.Columns.Add("Checkpoint " + (i+1).ToString());
+                if (first)
+                {
+                    t.Columns.Add("Global");
+                    first = false;
+                }
+                else
+                {
+                    t.Columns.Add("Checkpoint " + (i + 1).ToString());
+                }
             }
 
             // current point
             object[] row = new object[t.Columns.Count];
             row[0] = "PointIdentifier";
-            for (int i = _gameSave.Checkpoints.Count - 2; i >= 0; i--)
+            for (int i = _gameSave.Checkpoints.Count - 1; i >= 0; i--)
             {
-                row[_gameSave.Checkpoints.Count - i-1] = _gameSave.Checkpoints[i].PointIdentifier;
+                row[_gameSave.Checkpoints.Count - i] = _gameSave.Checkpoints[i].PointIdentifier;
             }
             t.Rows.Add(row);
 
             // flags
-            foreach (var flag in _gameSave.Data.flags)
+            foreach (string flag in Regex.Split(Resources.FlagList, "[\r\n]+"))
```

Comment:
Please use the following code
foreach(var flag in Resources.FlagList.Trim().Split(new [] { "\r\n" }, StringSplitOptions.RemoveEmptyEntries))
We do not need to use regular expressions.

Reply:
Right, thank you. :)